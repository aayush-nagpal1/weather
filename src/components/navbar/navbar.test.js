import React from 'react';
import Navbar from './navbar';
import renderer from 'react-test-renderer';

describe("Navbar Component", ()=>{
  
  
  it('navbar renders correctly', () => {
    const tree = renderer
      .create(<Navbar />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
})
