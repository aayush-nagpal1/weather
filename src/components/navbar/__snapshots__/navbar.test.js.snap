// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component 1 navbar renders correctly 1`] = `
<div>
  <div
    className="container"
  >
    <div
      className="row"
    >
      <div
        className="col-md-12"
      >
        <nav
          className="navbar navbar-expand-lg navbar-light bg-light"
        >
          <a
            className="navbar-brand"
            href="#"
          >
            Weather App
          </a>
          <button
            aria-controls="navbarSupportedContent"
            aria-expanded="false"
            aria-label="Toggle navigation"
            className="navbar-toggler"
            data-target="#navbarSupportedContent"
            data-toggle="collapse"
            type="button"
          >
            <span
              className="navbar-toggler-icon"
            />
          </button>
          <div
            className="collapse navbar-collapse"
            id="navbarSupportedContent"
          />
        </nav>
      </div>
    </div>
  </div>
</div>
`;

exports[`Navbar Component navbar renders correctly 1`] = `
<div>
  <div
    className="container"
  >
    <div
      className="row"
    >
      <div
        className="col-md-12"
      >
        <nav
          className="navbar navbar-expand-lg navbar-light bg-light"
        >
          <a
            className="navbar-brand"
            href="#"
          >
            Weather App
          </a>
          <button
            aria-controls="navbarSupportedContent"
            aria-expanded="false"
            aria-label="Toggle navigation"
            className="navbar-toggler"
            data-target="#navbarSupportedContent"
            data-toggle="collapse"
            type="button"
          >
            <span
              className="navbar-toggler-icon"
            />
          </button>
          <div
            className="collapse navbar-collapse"
            id="navbarSupportedContent"
          />
        </nav>
      </div>
    </div>
  </div>
</div>
`;
